app/uk/gov/hmrc/gform/models/FieldValueMaker.scala

```diff
@@ -60,22 +67,28 @@ class FieldValueMaker(json: JsValue) {
   lazy val infoText: Option[String] = (json \ "infoText").asOpt[String]
   lazy val infoType: Option[String] = (json \ "infoType").asOpt[String]
   lazy val shortName: Option[String] = (json \ "shortName").asOpt[String]
-  lazy val repeatsMax: JsResult[Option[Int]] = (json \ "repeatsMax").validateOpt[Int]
-  lazy val repeatsMin: Option[Int] = (json \ "repeatsMin").asOpt[Int]
```

Comment:
There are many ways to skin a cat :-), I think this solution and mine work fine. I don't see the need to have the Opt at this level, but if kittens could die if we don't have Opt at this level, here's the other part of my idea:

lazy val repeatsMax: Opt[Option[Int]] = handleJsonValidationResult( (json \ "repeatsMax").validateOpt[Int] )
lazy val repeatsMin: Opt[Option[Int]] = handleJsonValidationResult( (json \ "repeatsMin").validateOpt[Int] )